### DISCUSSION TITLE
When Forward Authentication is enabled, allow custom logout URL

### DISCUSSION THREAD

**Discussion by DDriggs00** [navidrome/navidrome#2830] [Ideas]
When the `ND_REVERSEPROXYUSERHEADER` environment variable is set, it allows navidrome to use forwarded proxy authentication from an external source, such as authentik or authelia, but breaks the logout button. Fixing the logout button is fairly simple, as most (if not all) forward authentication providers have a logout url that you can insert in lieu of the default logout mechanism. For this to work, all that is needed is to add support for a new environment variable (`ND_REVERSEPROXYLOGOUTURL`), and set the logout url to redirect there.

**Comment by deluan** [comment]
This has been discussed before, but I couldn't find the GH issue for it. Anyway, I agree this should be a very straightforward implementation. I'd gladly accept a PR implementing this :) but for now it is not in my priority list, as I'm focused in the new scanner (#2709) and the UI refactoring.

**Comment by ninolt** [comment]
Hello, any news regarding this suggestion ?

**Comment by helloclanker** [comment]
I may take a swing at this if you'd still accept a PR. Ran into this issue on my own setup with Authelia. Easy enough to logout via the auth URL but it was confusing not seeing a logout option inside Navidrome.

**Comment by deluan** [comment]
This was implemented in https://github.com/navidrome/navidrome/pull/5074, will be part of next release.
